sleap_nn/training/lightning_modules.py

```
@@ -584,3 +584,1300 @@ def validation_step(self, batch, batch_idx):
+
+            self.log(
+                f"train_loss_on_head_{d_num}",
+                curr_loss,
+                prog_bar=True,
+                on_step=False,
+                on_epoch=True,
+                logger=True,
+            )
+
+        self.log(
+            f"train_loss",
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)